app/presenters/summary/components/amount_and_frequency_answer.rb

```diff
@@ -7,7 +7,7 @@ def answer_text
         simple_format(
           I18n.t(
             'summary.dictionary.amount_and_frequency_answer',
-            amount: number_to_currency(value.amount),
```

Comment:
Number to currency has changed the way it handles big decimals and was failing to format the Pence types.